src/util.ts

```diff
@@ -81,15 +85,16 @@ export function getPropertyMap(
       member.type === 'PropertyDefinition' &&
       member.static &&
       member.key.type === 'Identifier' &&
-      member.key.name === 'properties' &&
-      member.value?.type === 'ObjectExpression'
+      member.key.name === 'properties'
     ) {
-      for (const prop of member.value.properties) {
+      const arg = member.value as ESTree.ObjectExpression;
```

Comment:
did you roll this back on purpose?

in master i got rid of the cast, thats what the additional condition is for in the `if` above. it should still be that way really

Reply:
huh, this was not on purpose, I guess something went wrong while merging. I'll take a look and fix it

Reply:
fixed :)